src/smt/theory_str_noodler/formula_preprocess.h

```diff
@@ -351,12 +352,13 @@ namespace smt::noodler {
 
 
     public:
-        FormulaPreprocessor(Formula conj, AutAssignment ass, std::unordered_set<BasicTerm> lv, const theory_str_noodler_params &par) :
+        FormulaPreprocessor(Formula conj, AutAssignment ass, std::unordered_set<BasicTerm> lv, const theory_str_noodler_params &par, std::unordered_set<BasicTerm> conversion_vars) :
```

Comment:
It is gonna be copied anyway, so it is better to keep it as copy parameter (so that c++ can use `move` semantics). Usually in constructors you want to pass as a copy and not reference.

Reply:
Ok